test/cache.js

```diff
@@ -895,9 +895,6 @@ test('supports matching using Vary header', t => {
   })
 })
 
-test('supports range caching (partial requests)')
```

Comment:
These todo tests are probably best to leave around?  I know they're noisy, but they're helpful to ensure we don't forget about important use cases when we do get time to bump up the test coverage in this repo.  (Though, I guess to be fair they are less useful in v5 than in latest, so removing on this branch is probably fine.)